frontend/src/App.js

```diff
@@ -4,15 +4,21 @@ import React from 'react';
 import { connect } from 'react-redux';
 
 import './App.css';
-
+import { L10nState } from 'core/l10n';
+import { LocaleState } from 'core/locales';
```

Comment:
These are Flow types and thus should be imported like so: 

```js
import type { L10nState } from 'core/l10n';
```

Also, I try to keep some structure to the imports, even though I haven't documented it yet. Here's what it is:

```js
import 'libraries';

import 'css';

import 'core and modules';

import 'local files';

import type 'types';
```